style.css

```diff
@@ -24,27 +17,36 @@ body {
     font-size: 20px;
     color: #929292;
     font-style: normal;
-    margin: 0px 70px 35px;
 }
 
-.search-input {
-    text-transform: initial;
-    color: #929292; 
-    align-content: center;
+form {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
 }
 
-input {
-    /* margin: 25px 10px 25px; */
+.main-content input.search-input {
```

Comment:
@ChrysaLynn 
I consolidated the styling here to have it be in quite a specific selector. It's the same one as before.